definitions/ext-dell_compellent/golden_metrics.yml

```diff
@@ -0,0 +1,24 @@
+diskCount:
+  title: Disk Count
+  unit: COUNT
+  query:
+    select: uniqueCount(name)
+    from: compellent_disks
+  displayAsValue: true
+
+failedDriveCount:
+  title: Failed Drive Occurences
+  unit: COUNT
+  query:
+    select: count(status_code) as 'Drive Failures'
+    from: compellent_disks
+    where: status = 'Down'
+  displayAsValue: true
+
+usedCapacityPercent:
```

Comment:
this will be the name to query by, for example:

`select newrelic.goldenmentric.ext.dell_compellent.usedCapacityPercent FROM Metric` not sure if we want to go with that or without the percent part

Reply:
thats fine